js/okx.js

```diff
@@ -1965,11 +1965,12 @@ module.exports = class okx extends Exchange {
         /**
          * @method
          * @name okx#createOrder
-         * @description create a trade order
+         * @description create a trade order *for margin buy orders, amount units are in quote currency*
+         * @see https://www.okx.com/docs-v5/en/#rest-api-trade-place-order
          * @param {string} symbol unified symbol of the market to create an order in
          * @param {string} type 'market' or 'limit'
          * @param {string} side 'buy' or 'sell'
-         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float} amount how much of currency you want to trade in units of base currency, *for margin buy orders, units are in quote currency*
```

Comment:
shouldn't we do the `create market buy requires price` thing then?

Reply:
Why would we? It's not required.